sql/mysql_db_structure.sql

```diff
@@ -37,7 +37,7 @@ CREATE TABLE `balloon` (
 -- Table structure for table `clarification`
 --
 CREATE TABLE `clarification` (
-  `clarid` int(4) unsigned NOT NULL AUTO_INCREMENT COMMENT 'Unique ID',
+  `clarid` int(4) unsigned NOT NULL AUTO_INCREMENT COMMENT 'Unique clarification ID',
```

Comment:
I changed the comment of this, and other primary keys, to not be "Unique ID", because if they are used as a foreign key in another table, Doctrine will reuse the comment. Sometimes it is possible to add an `@ORM\Column` to that table to overwrite this behaviour, but not in the case of ManyToMany associations or when the foreign key is part of the primary key.

We could even think about changing the comment from "Unique X ID" to "X ID", as the ID is not really unique in the table where it is a foreign key. Any thoughts?

p.s.: I never ever have even read the MySQL field comments. Especially now that we have an ORM, IMHO it is not that useful, but others might think otherwise.

Reply:
I would really prefer to keep the SQL comments, and find it a bit silly that Doctrine reuses those comments blindly in FKs. But if it does so, I'm fine with renaming those to "X ID".